dashboard/app/models/user.rb

```diff
@@ -1323,13 +1323,7 @@ def teacher?
   end
 
   def verified_teacher?
```

Comment:
We are still using this method but its only to check if someone specifically has this permission

Reply:
Maybe add a header comment that indicates how this is different from `verified_instructor?` and that, in most cases, the caller should be calling `verified_instructor?` instead.  Also, might be helpful to add a header comment to `verified_instructor?`.  (I'm just imagining someone seeing these two methods in the future and wondering which they should use.)

Also, mostly curious, what's a scenario where we'd want to check for specifically this permission?